docs/section19/19-11.md

```diff
@@ -0,0 +1,41 @@
+# Section 19 - Rule 19-11         
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** False    
+**Rule ID:** 19-11         
+**Rule Description:** For systems that serve computer rooms, if the  baseline system is HVAC System 11, it shall include an integrated fluid economizer meeting the requirements of Section 6.5.1.2 in the baseline building design.  
+
+**Rule Assertion:** Options NOT_APPLICABLE/UNDETERMINED      
+**Appendix G Section:** G3.1.2.6.1        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. Baseline HVAC systems that serve computer rooms and are of system type 11.    
+ 
+
+**Function Calls:**  
+1. get_baseline_system_types()  
+2. baseline_system_type_compare()     
+
+
+## Rule Logic:   
+**Applicability Check 1 - Check if there are any Applicable Baseline System Types in the Project at All**   
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_11]`  
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMR is modeled with at least one air-side system that is of type 11, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):` 
+    This segment identifies applicable HVAC system types (outcome for inapplicable system types will be NOT_APPLICABLE) 
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE outcome for the hvac system:  `If baseline_system_type_compare(sys_type, HVAC_SYS.SYS_11, false) == TRUE: `     
```

Comment:
There is FluidLoopDesignAndControl.has_integrated_waterside_economizer element that you should check. If it's a yes, then the outcome should be per Case 1. If it's a no, the outcome should be FAIL.

Reply:
@mkarpman This is non primary so we do not have a fail outcome only UNDETERMINED. We collectively decided on this because it was not clear which loop to check.

Reply:
Leave as is per phone call. Do not add.